pkgs/by-name/bi/bitcraze-udev-rules/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "bitcraze-udev-rules";
+  version = "0.1.23";
```

Comment:
I'm not entirely sure about versioning this against an upstream we don't use.

Would it be possible to create the udev rules with some fancy `sed` script in the bitcraze package itself? The README seems like it seldom gets updated, and if the sed script breaks in an update it seems like would be a good indicator that something might've changed about the udev rules as well?